sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/WritableColumnVector.java

```diff
@@ -443,6 +444,8 @@ public UTF8String getUTF8String(int rowId) {
     }
   }
 
+  public abstract ByteBuffer getBytesUnsafe(int rowId, int count);
```

Comment:
nit: maybe add a few comments here

Reply:
I agree, the method is misleading since there is a memory copy involved, it is just does not call System.arraycopy in OnHeapColumnVector.

Reply:
Yea I think we can use `getByteBuffer`. the "unsafe" here is a bit confusing.